README.md

```diff
@@ -324,7 +324,9 @@ Note: Only one of `token` and `bot` must be used.
 
 ### How it's made?
 
-There is a python script called `extractor.py`, that scrapes the Telegram Bot API documentation and prints to the stdout the lines needed to create all the methods and models, this auto generated lines uses two macros defined on `lib/ex_gram/macros.ex`: `method` and `model`.
+There is a python script called `extractor.py`, it uses the (`telegram_api_json`)[https://github.com/rockneurotiko/telegram_api_json] project that scrapes the Telegram Bot API documentation and provides a JSON with all the information, check the project description if you want to create your own projects that uses an standarized file to auto-generate the API.
```

Comment:
The URL will be displayed here:
<img width="940" alt="image" src="https://user-images.githubusercontent.com/9699815/78129355-c36d2000-7417-11ea-990f-7bbcd3f71cc7.png">

I suggest you remove the inverted commas from `telegram_api_json`:
```
... (telegram_api_json)[https://github.com/rockneurotiko/telegram_api_json] ...
```

Reply:
Thanks! The problem was the order of brackets, it's `[]()` and not `()[]` :joy: